File: dotnet/cafdemalihapa/Applications/IdeManagement/ProcessInformationGroup.cs
namespace cafdemalihapa.Applications.IdeManagement
{
    public class IdeProcessInformationGroup
    {
        public List<IdeProcessInformation>? Group { get; set; } = [];

        public void Add(IdeProcessInformation ideProcessInformation)
        {
            Group?.Add(ideProcessInformation);
        }
        public void AddInFront(IdeProcessInformation ideProcessInformation)
        {
            Group?.Insert(0, ideProcessInformation);
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Applications/IdeManagement/ProcessInformationGroup.fs
namespace cafdemwimu.console.Applications.IdeManagement

open System.Collections.Generic

[<AllowNullLiteral>]
type IdeProcessInformationGroup() =
    member val Group: List<IdeProcessInformation> = List<IdeProcessInformation>() with get, set

    member this.Add(ideProcessInformation: IdeProcessInformation) =
        this.Group.Add(ideProcessInformation)

    member this.AddInFront(ideProcessInformation: IdeProcessInformation) =
        this.Group.Insert(0, ideProcessInformation)
